README.rst

```diff
@@ -70,6 +70,15 @@ Requirements
 
 - Dependent libraries: requests, py7zr
 
+Documents
+---------
+
+There are versions of document
```

Comment:
This line doesn't quite look right to me. I would prefer it said this:

```diff
- Documents
+ Documentation
  -------------------

- There are versions of document
+ There are two versions of the documentation: 
```

Reply:
Also, above on line 5, we have the word "Documents" labelling a link to the documentation. Can we change that to the word "Documentation" as well?

Reply:
Thank you!

Actually there are all versions of documentation

Users can click bottom-left menu then find all versions;
![image](https://user-images.githubusercontent.com/123720/126572894-7d84da20-0054-44ed-9f35-d6b7dc73ff75.png)